database/things/java.pldb

```diff
@@ -457,6 +457,9 @@ stackOverflowSurvey
   medianSalary 51888
   fans 17222
   percentageUsing 0.35
+annualReports https://www.oracle.com/java/technologies/javase/jdk-relnotes-index.html releaseNotesUrl
+annualReports https://blogs.oracle.com/java/releaseNotesUrl officialBlogUrl
+annualReports https://dev.java/community/events/ eventsPageUrl
```

Comment:
Again, these lines have "releaseNotesUrl", "officialBlogUrl" and "eventsPageUrl" at  the end of their lines. Is this a problem with wrapping at the end of a line?